api/swagger-spec/swagger.yml

```diff
@@ -9,6 +9,15 @@ consumes:
 schemes:
   - http
 paths:
+  /ping:
```

Comment:
"/ping" seems a little too general for something as specific as "check to see if the `workflow-manager` component has a live, working connection to its `versions` API. Either of these would be preferable:

- a more descriptive route endpoint, like "/ping-versions-api"
- a hierarchical backend ping structure that accommodates future add'l backend ping checks, e.g., "/apideps/versions/ping"